DQM/L1TMonitor/src/L1TdeStage2CaloLayer2.cc

```diff
@@ -1012,568 +1012,571 @@ bool L1TdeStage2CaloLayer2::compareSums(
   double dataPhi = 0;
   double emulPhi = 0;
 
-  l1t::EtSumBxCollection::const_iterator dataIt = dataCol->begin(currBx);
+  l1t::EtSumBxCollection::const_iterator dataIt;
   l1t::EtSumBxCollection::const_iterator emulIt = emulCol->begin(currBx);
 
   // if either data or emulator collections are empty, or they have different
   // size, mark the event as bad (this should never occur in normal running)
```

Comment:
Hi @vukasinmilosevic 
can you please also update the comment corresponding to the changed code?

Reply:
Ah yes. Will do it now.